lib/fluent/plugin/in_groonga.rb

```diff
@@ -132,7 +131,6 @@ def configure(conf)
 
       def start
         listen_socket = TCPServer.new(@bind, @port)
```

Comment:
I've tried to work with multi worker environment with the following patches and confirm to pass test cases:
https://github.com/groonga/fluent-plugin-groonga/compare/use-v0.14-api-in-groonga...cosmo0920:adopting-multi-workers